helpers/authentication.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+import (
+	"log"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+// EncryptPassword encrypt given password, returns encrypted password and error
+func EncryptPassword(password string) (string, error) {
```

Comment:
@carryall It is not  encrypting, it is actually hashing. So how about rename it to `HashPassword`?